app/packs/src/components/WelcomeMessage.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Jumbotron } from 'react-bootstrap';
+import ReactMarkdown from 'react-markdown';
+
+function Message(props) {
+  const input = props.post;
+  if (input != null && input !== undefined)  {
+    return <Jumbotron> <ReactMarkdown children={input} /> </Jumbotron>;
+  }else{
+    return null;
+  }
+}
+
+class WelcomeMessage extends Component {
+  state = {
+    post: null
+  }
+  componentDidMount () {
+    fetch('welcome-message.md')

```

Comment:
@NishaSharma14  you fetch the wrong file name.

Reply:
filename is correct. But this file should be optional and its in gitignore.